lib/WebRtcPeer.js

```diff
@@ -132,21 +132,21 @@ function bufferizeCandidates(pc, onerror) {
   return function (candidate, callback) {
     callback = callback || onerror;
     switch (pc.signalingState) {
-    case 'closed':
-      callback(new Error('PeerConnection object is closed'));
-      break;
-    case 'stable':
-      if (pc.remoteDescription) {
-        pc.addIceCandidate(candidate, callback, callback);
+      case 'closed':
+        callback(new Error('PeerConnection object is closed'));
         break;
+      case 'stable':
+        if (pc.remoteDescription) {
+          pc.addIceCandidate(candidate, callback, callback);
+          break;
 
-      }
-      default:
-        candidatesQueue.push({
-          candidate: candidate,
-          callback: callback
+        }
```

Comment:
I know wrong indentation can cause some severe OCD bursts to some devs (me included), but I'd rather avoid whitespace changes for a future addition of Prettier, when a full reformat will be applied to all code. Could you please revert all these lines? Otherwise we could just merge https://github.com/Kurento/kurento-utils-js/pull/30 which seems to do the same